app.amsterdam.json

```diff
@@ -128,5 +131,8 @@
   },
   "azure": {
     "connectionString": "InstrumentationKey=c39a786a-82ac-471d-a0c6-feb41eb4967b;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/"
+  },
+  "frontPageAlert": {
```

Comment:
Because it is optional you can remove it here.

Reply:
I've removed it, but I personally would have liked to have it there. Just so that next time I only have to add text.